gravitee-apim-console-webui/src/management/configuration/configuration.route.ts

```diff
@@ -825,6 +825,24 @@ function configurationRouterConfig($stateProvider: StateProvider) {
         },
       },
     })
+    .state('management.settings.notification-settings', {
+      url: '/notification-settings',
+      component: 'environmentNotificationSettingsList',
+      data: {
+        docs: {
+          page: 'management-configuration-notification-settings',
```

Comment:
```suggestion
          page: 'management-configuration-notifications',
```
Same as previous routing. We d'ont change the documentation config for now

Reply:
Done